### PR TITLE
Refactors schedule pages

### PR DESCRIPTION
**Pull request by jefftriplett**
This moves from a hacky post model to utilizing our own schedule collection instead.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -57,6 +60,11 @@ defaults:
       path: ""
     values:
       image: /static/2017-logos/squares-and-social/djangocon-square.png
+  # - scope:
+  #     path: ""
+  #     type: scheule
```
Comment (FlipperPA) [inline _config.yml]: Typo: `schedule`?

**Comment by FlipperPA** [comment]
Assuming we get that typo fixed, it looks great to me. Thanks for checking!

File: _config.yml (hunk 1 above)
Comment (jefftriplett) [reply _config.yml]: Thanks, it didn't work anyways, so I removed it.

File: _schedule/talks/2016-07-18-09-00-keynote-1.md [hunk 2]
```diff
@@ -4,20 +4,21 @@ category: talk
...
 permalink: /talks/keynote-1/
-presenters: Saron Yitbarek, CodeNewbie
-photo_url: https://placem.at/people?w=400&h=400&random=3
 published: true
 sitemap: false
 room: Auditorium (G-06)
 summary: ''
 title: Keynote
 track: ''
+presenters:
```
Comment (jefftriplett) [inline _schedule/talks/2016-07-18-09-00-keynote-1.md:13]: @FlipperPA this reflects the changes we'll need to make in the data exporter. This allows us to have multiple presenters which we can add post-export by hand.

**Pull request merged**